package.json

```diff
@@ -6,17 +6,28 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "format": "prettier --check --ignore-path .gitignore .",
```

Comment:
let's remove the `--ignore-path` flag and add a new file called `.prettierignore` that lists all the stuff we want to ignore